drop-in/src/main/java/com/adyen/checkout/dropin/DropInService.kt

```diff
@@ -37,9 +38,9 @@ abstract class DropInService : BaseDropInService(), DropInServiceContract {
         onAdditionalDetails(actionComponentData)
     }
 
-    final override fun requestBalanceCall(paymentMethodData: PaymentMethodDetails) {
+    final override fun requestBalanceCall(paymentComponentData: PaymentComponentData<GiftCardPaymentMethod>) {
```

Comment:
Minor: is there any reason to use `PaymentComponentData<GiftCardPaymentMethod>` instead of `GiftCardComponentState`?

Reply:
Changed it to `PaymentComponentState<*>`